### PR TITLE
crackling/kipper: Add back libcneapiclient

### PR DESCRIPTION
**Pull request by tdm**
Change-Id: I43742747586cd132584d5cf60f31645f674f0c20

### CODE REVIEW

File: kipper/kipper-vendor.mk [hunk 1]
```diff
@@ -148,6 +148,7 @@ PRODUCT_COPY_FILES += \
     vendor/wileyfox/kipper/proprietary/vendor/lib/libchromatix_ov8858_q8v19w_snapshot.so:system/vendor/lib/libchromatix_ov8858_q8v19w_snapshot.so \
     vendor/wileyfox/kipper/proprietary/vendor/lib/libchromatix_ov8858_q8v19w_video_nonHD.so:system/vendor/lib/libchromatix_ov8858_q8v19w_video_nonHD.so \
     vendor/wileyfox/kipper/proprietary/vendor/lib/libchromatix_ov8858_q8v19w_zsl.so:system/vendor/lib/libchromatix_ov8858_q8v19w_zsl.so \
+    vendor/wileyfox/kipper/proprietary/vendor/lib/libcneapiclient.so:system/vendor/lib/libcneapiclient.so \
```
Comment (zwliew) [inline kipper/kipper-vendor.mk]: You're adding the wrong path

Comment (zwliew) [inline kipper/kipper-vendor.mk:151]: This is the wrong path

File: crackling/crackling-vendor.mk [hunk 2]
```diff
@@ -151,6 +151,7 @@ PRODUCT_COPY_FILES += \
     vendor/wileyfox/crackling/proprietary/vendor/lib/libchromatix_s5k5e2_olq5f19_snapshot.so:system/vendor/lib/libchromatix_s5k5e2_olq5f19_snapshot.so \
     vendor/wileyfox/crackling/proprietary/vendor/lib/libchromatix_s5k5e2_olq5f19_video_hd.so:system/vendor/lib/libchromatix_s5k5e2_olq5f19_video_hd.so \
     vendor/wileyfox/crackling/proprietary/vendor/lib/libchromatix_s5k5e2_olq5f19_video.so:system/vendor/lib/libchromatix_s5k5e2_olq5f19_video.so \
+    vendor/wileyfox/crackling/proprietary/vendor/lib/libcneapiclient.so:system/vendor/lib/libcneapiclient.so \
```
Comment (zwliew) [inline crackling/crackling-vendor.mk:154]: This is the wrong path as well

**Review by zwliew — CHANGES_REQUESTED** [review]

**Comment by tdm** [comment]
Sorry, how is this the wrong path?

**Comment by zwliew** [comment]
Oh nevermind, I didn't realise that the 32-bit libcneapiclient lib was already present in the vendor tree but wasn't being installed.

**Comment by tdm** [comment]
@zwliew yes, that was an oversight when Mike did the 14.1 changes to crackling.  Then he duplicated the work over to kipper.

**Comment by mikeNG** [comment]
#7 is the correct fix, I pushed the wrong blobs :/

**Pull request closed without merging**